aj-may/eybc-bip
# feat: Drafts page implementation

### Related Ticket



https://eyblockchain.atlassian.net/browse/BIPT-42

### Description

Created a lazy implementation using a similar schema to the currently existing "All" page. I submit this PR so I can unblock the development of other features (via api implementation) but with full intentions to refactor most of front-end code into more concise, reusable components in the future. 

### QA Instructions

-[ ] checkout to `meag/draft-page-refactor`
-[ ] npm run dev
-[ ] toggle between the `All` and `Drafts` tabs to see the filtered vs. non-filtered pages

### Checklist



- [x] I have reviewed the code changes myself
- [x] I have removed unused code (e.g., `console.logs`, commented out blocks, etc.)
- [x] I have made all necessary changes to the documentation.
- [x] Checked my PR title name is correct.
